xbmc/pvr/dialogs/GUIDialogPVRChannelManager.h

```diff
@@ -46,7 +46,7 @@ namespace PVR
   private:
     void Clear();
     void Update();
-    void SaveList();
+    void SaveList(bool bPrompt = false);
```

Comment:
I'm not a fan of bool parameters, because the calling code is not self explaining. I would leave `SaveList()` implementation as it is before this PR and introduce a new method `PromptAndSaveList()`, prompting and calling `SaveList()`, depending on the user decision.

Reply:
I like that method better too, updated!